index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prangon's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logo-placeholder">
+                <div class="my-logo">
+                    PG
+                </div>
+            </div>
+            <div class="menu">
+                <div class="icon">
+                    <div class="union">
+                        <button class="nav-button"><img src="images/Union.png" alt="menu-image" class="menu-image"></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <div class="container">
+        <div class="headline">
+            <div class="primary-text">
+                I'm Prangon<br>Glad to see you!
+            </div>
+            <div class="description">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
```

Comment:
- I think this text should be a paragraph and in a `<p>` tag you can replace `<div class="description">` to `<p class="description">` ✔️